test/board_test.rb

```diff
@@ -55,19 +55,28 @@ def test_placing_ship
     assert cell_3.ship == cell_2.ship
   end
 
-
   def test_for_overlapping_ship
     @board.place(@cruiser, ["A1", "A2", "A3"])
+    
     refute @board.valid_placement?(@submarine, ["A1", "B1"])
-  
   end
 
   def test_rendering_the_board
     @board.place(@cruiser, ["A1", "A2", "A3"])
-
-    assert_equal "  1 2 3 4 \nA . . . . \nB . . . . \nC . . . . \nD . . . . \n", @board.render
-
-    #assert_equal "  1 2 3 4 \nA S S S . \nB . . . . \nC . . . . \nD . . . . \n", @board.render(true)
+    expected = "  1 2 3 4 \n" +
+               "A . . . . \n" +
+               "B . . . . \n" +
+               "C . . . . \n" +
+               "D . . . . \n"
+
+    assert_equal expected, @board.render
```

Comment:
This test was not showing the entire grid. We learned that it wasn't failing where we thought it was and the missing lines may not have been shown by minitest because we were meant to be focusing on a different area of the grid. Turns out we were missing a space. Helpful to use the string representation of the grid in the test instead of the one provided in the interaction pattern.